gensim/utils.py

```diff
@@ -1022,7 +1014,20 @@ def pyro_daemon(name, obj, random_suffix=False, ip=None, port=None):
             daemon.requestLoop()
 
 
-if HAS_PATTERN:
+def has_pattern():
+    """
+    Function to check if there is installed pattern library
+    """
+    pattern = False
+    try:
+        from pattern.en import parse
+        pattern = True
+    except ImportError:
+        logger.warning("Pattern library is not installed.")
+    return pattern
+
+
+if has_pattern():
```

Comment:
Hmm, do we even need this `if`? When users try to call `lemmatize`, and pattern is not installed, they'll get a clear, standard `ImportError`.

I'm not sure why this `if` is here, I don't think it belongs any more, conceptually. But maybe I'm missing something.